index.html

```diff
@@ -376,17 +391,117 @@ <h3>
 Deactivate (Revoke)
           </h3>
           <p>
-To delete the DID document, the <code>did.json</code> has to be removed or has
-to be no longer publicly available due to any other means.
+To delete the DID document, the associated file or files have to be removed or
+made no longer publicly available due to any other means.
           </p>
         </section>
       </section>
 
+      <section>
+      <h2>
+Webserver Configuration
+      </h2>
+      <h3>
+MIME Types
+      </h3>
+      <p>
+Webservers MUST be configured to associate proper content types with registered
```

Comment:
I would prefer to see `application/json` as the only defined format first... and then attempt to address these other representations separately... too many changes in this PR.

Reply:
In #47, I reduced to `application/json` and `application/did+json` to preserve the logic of shifting to index files instead of hardcoded `/did.json`.

I also removed references to JSON-LD.

This presents us with a path forward to comfortably support other representations in the future without embracing the complexity today.